cf-cli/Dockerfile

```diff
@@ -28,3 +29,10 @@ RUN curl -L -o "${CF_AUTOPILOT_TEMPFILE}" \
   && chmod +x "${CF_AUTOPILOT_TEMPFILE}" \
   && cf install-plugin -f "${CF_AUTOPILOT_TEMPFILE}" \
   && rm "${CF_AUTOPILOT_TEMPFILE}"
+
+RUN apk add --update wget ca-certificates \
```

Comment:
ca-certificates is already installed and probably you should not remove it at the end

Reply:
also curl is available so, use that instead of wget